src/app.py

```diff
@@ -107,5 +107,29 @@ def add_course():
     return redirect(url_for("start"))
 
 
+# router to course detailed page
```

Comment:
Could you move this comment on top of the function? Maybe start with Captial as well